### PR TITLE
Add macOS support on AppVeyor

### PR DESCRIPTION
**Pull request by YannickJadoul**
Stealing @mayeut's branch that was mentioned in #225; curious to see if it now works :-)

### CODE REVIEW

**Comment by YannickJadoul** [comment]
Hmmm, not working yet, it seems, but macOS is available now.

@mayeut, do you know what to do? Or if you don't have time, do you mind if I push to your branch?

**Comment by mayeut** [comment]
> @mayeut, do you know what to do? Or if you don't have time, do you mind if I push to your branch?

I don't really know what to do about this SSL error on macOS...
You can push to this branch if you want to. Let's just not rebase/force-push anything just yet if we'll both be using it at some point.

**Comment by mayeut** [comment]
Seems there are some issues with python on macOS:
https://github.com/appveyor/ci/issues/3213

**Comment by YannickJadoul** [comment]
> Seems there are some issues with python on macOS:
> [appveyor/ci#3213](https://github.com/appveyor/ci/issues/3213)

Argh. Guess we'll just have to wait for that to get fixed, then?

**Comment by mayeut** [comment]
@YannickJadoul, @joerick,

I think it's ready to merge !

AppVeyor fixed there issues on macOS and I updated the docs & examples (unless I missed something else in the docs).

File: examples/appveyor-minimal.yml [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+      APPVEYOR_JOB_NAME: "python37-x64-vs2015"
+    - APPVEYOR_BUILD_WORKER_IMAGE: macos-mojave
+      APPVEYOR_JOB_NAME: "python37-x64-macos-mojave"
+
+stack: python 3.7
+
+init:
+- cmd: set PATH=C:\Python37;C:\Python37\Scripts;%PATH%
+
+install: python -m pip install cibuildwheel==1.1.0
```
Comment (joerick) [inline examples/appveyor-minimal.yml]: If this is just `python`, will it run with Python3 on macOS? Normally `python` means Python 2 on macOS, unless Appveyor is doing something clever. I guess if it builds, that answers our question!

Comment (mayeut) [reply examples/appveyor-minimal.yml]: We're using the python stack feature of AppVeyor which setups a virtualenv (macOS/Linux only) so we're using python 3.7

Comment (mayeut) [reply examples/appveyor-minimal.yml]: PS: it builds, https://ci.appveyor.com/project/joerick/cibuildwheel/builds/31310546
I've rebased the branch on latest master, that's why the status is pending.

Comment (joerick) [reply examples/appveyor-minimal.yml]: Cool. Thanks!

**Pull request merged**